test/james.crypto.md5.tests.pas

```diff
@@ -76,8 +82,8 @@ procedure TMD5StreamTest.StreamFromMemory;
 initialization
   RegisterTestS(
     'Crypto', [
-      TMD5HashTest,
-      TMD5StreamTest
+      TMD5HashTest{$IFNDEF FPC}.Suite{$ENDIF},
```

Comment:
@nunopicado I think this is Ok for now, but we should to create a single unit to rewrite all this:

```pascal
  {$IFDEF FPC}
     fpcunit, testregistry
   {$ELSE}
     TestFramework
   {$ENDIF};
//...
TMD5HashTest{$IFNDEF FPC}.Suite{$ENDIF},
```
To something like this:

```pascal
uses
   // ...
  James.UnitTest;
  //...
  James.UnitTest.RegisterTests(
    'Crypto', [
      TMD5HashTest,
      TMD5StreamTest,
      TMD5HashTest
    ]
  )
```